### PR TITLE
Fixed the installation of npm on Linux

### PR DESCRIPTION
**Pull request by omichel**
This PR makes that travis install will the `npm` package the same was we do it manually following the new explanations here: https://github.com/omichel/webots/wiki/Linux-installation.

It is simpler and greatly facilitates the build on snapcraft.io.

### CODE REVIEW

**Comment by omichel** [comment]
:+1:

**Comment by DavidMansolino** [comment]
(I just started the test including the compilation process)

**Comment by omichel** [comment]
Apparently ubuntu 16 (on travis) only has `libssl-dev`, but on ubuntu 16, `libssl-dev` doesn't conflict with `npm` which is fine this way.

👍

**Comment by DavidMansolino** [comment]
Ok thank you for the test. We should then also update: https://github.com/omichel/webots/wiki/Linux-installation

**Comment by omichel** [comment]
Correct. This is done now.
👍

**Comment by fabienrohrer** [comment]
👀

File: .travis.yml [hunk 1]
```diff
@@ -90,13 +90,14 @@ jobs:
             - libpci-dev
             - libgd-dev
             - libtiff5-dev
-            - libssh-dev
+            - libssh-gcrypt-dev
```
Comment (fabienrohrer) [inline .travis.yml]: I don't understand why we need only `libssh-gcrypt-dev` here and we ask users to install `libssh-dev` in the wiki. Isn't `libssh-gcrypt-dev` sufficient for every cases?

Comment (fabienrohrer) [reply .travis.yml]: Forget my comment `libssh-gcrypt-dev` is mentioned in the wiki.

**Pull request merged**